lib/fp.js

```diff
@@ -71,31 +66,28 @@ const zip = (
   return res;
 };
 
-const replicate = (
-  // Create array of replicated values
-  count, // number, new array length
-  elem // value to replicate
-  // Returns: array, replicated
-) => (Array.from({ length: count }, () => elem));
-
-const zipWith = (
-  // Zip arrays using specific function
-  fn, // function, for zipping elements with index i
-  ...arrays // array of arrays
-  // Returns: array
-  // Element with index i of resulting array is result
-  // of fn called with arguments from arrays
-) => zip(...arrays).map(args => fn(...args));
-
-const curryUntil = (
-  // Curry function until the condition is met
-  condition, // function, (argsI, argsParts) returns boolean
-  // argsI is arguments for i-th currying
-  // argsParts is array of args given for currying from first to i-th currying
-  fn, // function, which will be curried
-  ...args // array, arguments for fn
-  // Returns: function, curried
-) => {
+// Create array of replicated values
+//   count - number, new array length
+//   elem - value to replicate
+// Returns: array, replicated
+const replicate = (count, elem) => (Array.from({ length: count }, () => elem));
+
+// Zip arrays using specific function
+//   fn - function, for zipping elements with index i
+//   arrays - array of arrays
+// Returns: array
```

Comment:
Elaborate on the meaning of array (data in it) here please. (Also applies in other places)